### PR TITLE
69 create the area relations

### LINKED ISSUE #69: Create the A/REA relations
**Short description**
You have to create a api route which allow to create a A/REA relation with an user.
 
**Describe the solution you'd like**
This route should be set to POST. It's a you to precise which field is required.
It should create a value in UserHasActionReactions model.

### PR DESCRIPTION
**Pull request by ZQUEMA**
# Purpose

This PR add the possibility to create a A/REA relation.

Fixes #69 

## Added features

* It adds many dev routes to create some entities
  * User -> /api/dev/user/create 
  * Service -> /api/dev/user/service 
  * Action -> /api/dev/user/action 
  * Reaction -> /api/dev/user/reaction
  * Parameter -> /api/dev/user/parameter
* It add the route /api/area/create to create a A/REA relation

### CODE REVIEW

**Review by tbellicha — CHANGES_REQUESTED** [review]
Missing documentation on all implemented functions

File: server/server_app.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ passport.deserializeUser((id, done) => {
 })
 
 app.use(bodyParser.json())
-app.use(session({ secret: 'SECRET' }))
+app.use(session({ secret: 'SECRET', resave: false, saveUninitialized: false }))
```
Comment (LucasTesnier) [inline server/server_app.js:35]: Why did you updated that ?

File: server/server_app.js [hunk 2]
```diff
@@ -464,13 +412,146 @@ app.get(
   }
 )
 
+app.post('/api/dev/user/create', async (req, res) => {
+  console.log(req.body)
```
Comment (LucasTesnier) [inline server/server_app.js]: Why did you log the body ?

File: server/server_app.js [hunk 3]
```diff
@@ -464,13 +412,146 @@ app.get(
   }
 )
 
+app.post('/api/dev/user/create', async (req, res) => {
```
Comment (LucasTesnier) [inline server/server_app.js:421]: Documentation

**Review by LucasTesnier — CHANGES_REQUESTED** [review]
ANswer my question, remove useless login, document your code and please use the correct format (CF Discord channel)

**Review by TomDUVAL-MAHE — CHANGES_REQUESTED** [review]
Missing doc 
Bad format

File: server/server_app.js (hunk 1 above)
Comment (ZQUEMA) [reply server/server_app.js:35]: Don't know
It's may be the auto-format

**Review by TomDUVAL-MAHE — DISMISSED** [review]
Seems good to me

**Review by LucasTesnier — APPROVED** [review]

**Review by tbellicha — APPROVED** [review]

**Pull request merged**